pkg/proc/stack.go

```diff
@@ -211,6 +226,9 @@ func (it *stackIterator) Next() bool {
 	}
 
 	callFrameRegs, ret, retaddr := it.advanceRegs()
+	if it.err != nil {
```

Comment:
Why?

Reply:
Still wondering about this.

Reply:
Without, `TestFindReturnAddressTopOfStackFn` fails:

```
--- FAIL: TestFindReturnAddressTopOfStackFn (0.63s)
    support.go:268: enabling recording for github.com/go-delve/delve/pkg/proc_test.TestFindReturnAddressTopOfStackFn
    proc_test.go:793: expected error to be returned
```

Reply:
Is that it? If it's the only one check what returnAddress returns (especially the function the address belongs to), I think it's just that for some reason _main in rt0_linux_ppc64.s calls into runtime.rt0_go instead of jumping to it like the amd64 and arm64 do (I think `BR (CLR)` is the go assembler version of the branch to subroutine instruction but I'm not 100% sure)